### ISSUE TITLE
Chore: Add Link to 'About' in Landing Page Footer

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#342]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [x] Codebase
- [ ] User Interface
- [ ] Feature Request

## Description

We need to add a link to 'About' in landing page footer.

Link: https://openpedia.netlify.app/#about

**Comment by Bibek478** [comment]
Assign me this issue under jwoc24

**Comment by thatguywhocode** [comment]
Hey @Sriparno08 , assign me this task under JWOC.

**Comment by Sriparno08** [comment]
@thatguywhocode You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by Sneha123-zudo** [comment]
Please assign me this issue under JWOC'24

**Comment by sp-gits** [comment]
Assign me this issue , i can fix it

**Comment by thatguywhocode** [comment]
Hey @Sriparno08 , assign me this task under JWOC.

**Comment by Sriparno08** [comment]
@Sneha123-zudo You're assigned.

@thatguywhocode You've made the PR, but it's not merged yet. So, you are not considered for this issue.
